File: 0001 Code Chef/0021 Masterchief final/implementation/0006 php/master_chef.php
<?php
    $test = (int) fgets(STDIN);
    while($test > 0){
        $rank = (int) fgets(STDIN);
        if($rank <= 10 && $rank >= 1){
            echo "YES\n";
        }else{
            echo "NO\n";
        }
        $test = $test - 1;
    }
?>

File: 0001 Code Chef/0020 Masterchief final/implementation/0004 java/MasterChef.java
import java.util.Scanner;

public class MasterChef{
    public static void main(String[] args){
        Scanner console  = new Scanner(System.in);
        String stringInput;
        stringInput = console.nextLine().trim();
        int test = Integer.parseInt(stringInput);
        while(test > 0){
            stringInput = console.nextLine().trim();
            int rank = Integer.parseInt(stringInput);
            if(rank <= 10){
                System.out.println("YES");
            }else{
                System.out.println("NO");
            }
            test = test - 1;
        }
    }
}